src/bundle/Resources/public/js/scripts/sidebar/main.menu.js

```diff
@@ -0,0 +1,132 @@
+
+        parseMenuTitles();
+        setWidthOfSecondLevelMenu();
+    };
+    const setWidthOfSecondLevelMenu = () => {
+        if (!localStorage.getItem('secondLevelMenuWidth') || secondLevelMenuNode.classList.contains('ibexa-main-menu__navbar--hidden')) {
+            return;
+        }
+
+        const { secondLevelMenuWidth } = localStorage;
+        const secondLevelMenuListWidth = secondLevelMenuWidth - RESIZER_WIDTH;
+
+        secondLevelMenuNode.style.width = `${secondLevelMenuWidth}px`;
+        secondLevelMenuNode.querySelectorAll('.ibexa-main-menu__tab-pane .ibexa-main-menu__items-list').forEach((itemList) => {
+            itemList.style.width = `${secondLevelMenuListWidth}px`;
+        });
+        secondLevelMenuNode.classList.toggle(
+            'ibexa-main-menu__navbar--collapsed',
+            secondLevelMenuWidth <= SECOND_LEVEL_MANUAL_RESIZE_MIN_WIDTH
+        );
+    };
+    const toggleSecondLevelMenu = () => {
+        const newMenuWidth = secondLevelMenuNode.classList.contains('ibexa-main-menu__navbar--collapsed')
+            ? SECOND_LEVEL_EXPANDED_WIDTH
+            : SECOND_LEVEL_COLLAPSED_WIDTH;
+
+        localStorage.secondLevelMenuWidth = newMenuWidth;
+        setWidthOfSecondLevelMenu();
+    };
+    const parsePopup = (button) => {
+        const { popupTargetSelector } = button.dataset;
+        const popupNode = doc.querySelector(popupTargetSelector);
+
+        if (!popupNode) {
+            return;
+        }
+
+        const popup = new eZ.core.PopupMenu({
+            popupMenuElement: popupNode,
+            triggerElement: button,
+        })
+    };
+    const parseMenuTitles = () => {
+        firstLevelMenuNode.querySelectorAll('.ibexa-main-menu__item').forEach((item) => {
+            const label = item.querySelector('.ibexa-main-menu__item-text-column').textContent;
+
+            if (firstLevelMenuNode.classList.contains('ibexa-main-menu__navbar--collapsed')) {
+                item.setAttribute('title', label);
+            } else {
+                item.removeAttribute('data-original-title');
+            }
+
+            eZ.helpers.tooltips.parse(mainMenuNode);
+            item.removeAttribute('title');
+        });
+
+        
+        eZ.helpers.tooltips.hideAll();
+    };
+    const addResizeListeners = ({ clientX }) => {
+        resizeStartPositionX = clientX;
+        secondLevelMenuNode.classList.add('ibexa-main-menu__navbar--resizing');
+        secondMenuLevelCurrentWidth = secondLevelMenuNode.getBoundingClientRect().width;
+
+        doc.addEventListener('mousemove', triggerSecondLevelChangeWidth, false);
+        doc.addEventListener('mouseup', removeResizeListeners, false);
+    };
+    const removeResizeListeners = () => {
+        doc.removeEventListener('mousemove', triggerSecondLevelChangeWidth, false);
+        doc.removeEventListener('mouseup', removeResizeListeners, false);
+        secondLevelMenuNode.classList.remove('ibexa-main-menu__navbar--resizing');
```

Comment:
some consistency needed - move this line about `removeEventListeners` to match order of previous func